### PR TITLE
File step

### PR DESCRIPTION
**Pull request by soaresg**
I created a step where you can insert an image from the gallery or open the camera to take a picture. 

This step inserts in the JSON result the path of the image in the app cache as a string, and with that, the developer chooses where the image will be stored.

### CODE REVIEW

File: .fvm/fvm_config.json [hunk 1]
```diff
@@ -0,0 +1,4 @@
+{
+  "flutterSdkVersion": "3.0.5",
+  "flavors": {}
+}
```
Comment (Numoy) [inline .fvm/fvm_config.json:4]: Please remove the file from your commit

File: .fvm/flutter_sdk [hunk 2]
```diff
@@ -0,0 +1 @@
+/home/gabrielsoares/fvm/versions/3.0.5
```
Comment (Numoy) [inline .fvm/flutter_sdk:1]: Please remove the file from your commit

File: .gitignore [hunk 3]
```diff
@@ -1,4 +1,5 @@
 # Miscellaneous
+.gitignore
```
Comment (Numoy) [inline .gitignore:2]: Why should it be ignored?

File: lib/src/views/image_answer_view.dart [hunk 4]
```diff
@@ -0,0 +1,158 @@
...
+                ),
+                child: Row(
+                  mainAxisAlignment: MainAxisAlignment.spaceEvenly,
+                  crossAxisAlignment: CrossAxisAlignment.center,
+                  children: [
+                    ElevatedButton(
+                      onPressed: () {
+                        _optionsDialogBox();
+                      },
+                      child: Text('Image :'),
```
Comment (Numoy) [inline lib/src/views/image_answer_view.dart:76]: Can you please make the text configurable through the ImageAnswerFormat?

**Review by Numoy — CHANGES_REQUESTED** [review]
First, thank you for your contribution. This is heavily appreciated!

Please have a look at my comments.

File: .gitignore (hunk 3 above)
Comment (soaresg) [reply .gitignore]: Sorry for that, it was just one test that I make and forgot to take out. In the new commit, I take that out.

File: .fvm/flutter_sdk (hunk 2 above)
Comment (soaresg) [reply .fvm/flutter_sdk]: Folder removed from project.

**Pull request merged**